assets/Javascript files/script.js

```diff
@@ -0,0 +1,54 @@
+// Code for the Hackathon
```

Comment:
I'll be honest. I don't really have that many criticisms. I was genuinely impressed with your project. Maybe next time try to add a back end? otherwise, great job!

Reply:
Thank you!